meta/documents/changelog_en.md

```diff
@@ -1,5 +1,17 @@
 # Release Notes for IO
 
+## v5.0.24 (2021-03-08) <a href="https://github.com/plentymarkets/plugin-io/compare/5.0.23...5.0.24" target="_blank" rel="noopener"><b>Overview of all changes</b></a>
+
+### Changed
+
+- In the checkout, the existing shopping cart is no longer emptied, but replaced by a new empty one instead. This speeds up order creation especially for bigger shopping carts.
+
+### Fixed 
+
+- In case the routes /login and /register were not linked to a category, they were not available in the sitemap. This has been fixed.
```

Comment:
```suggestion
- In case the routes /login and /register were not linked to a category, they were not listed in the sitemap. This has been fixed.
```